package-lock.json

```diff
@@ -13,6 +13,10 @@
         "ajv-keywords": "^5.1.0",
         "dotenv": "^16.0.3",
         "express": "^4.18.2",
+        "dotenv": "^16.0.1",
+        "express": "^4.18.1",
```

Comment:
Repeated dependency declarations here, whoops. I am surprised npm did not error on this.

I am still trying to figure out why several `@types` dependencies are ending up outside the dev dependencies area when doing `npm install`.